mohawk/tests.py

```diff
@@ -656,6 +657,17 @@ def test_bewit(self):
         expected = '123456\\1356420707\\IGYmLgIqLrCe8CxvKPs4JlWIA+UjWJJouwgARiVhCAg=\\'
         eq_(b64decode(bewit).decode('ascii'), expected)
 
+    def test_bewit_with_binary_id(self):
+        # Check for exceptions in get_bewit call with binary id
+        binary_credentials = deepcopy(self.credentials)
+        binary_credentials['id'] = binary_credentials['id'].encode()
```

Comment:
What' does an empty encode() do? Could this be more explicit like `.encode('ascii')` ?